build-remote

```diff
@@ -204,7 +204,7 @@ def main():
                 shouldExit = True
             arch = os.path.basename(build.distro_arch_series_link)
             for f in build.getFileUrls():
-                if not f.endswith(".rootfs.tar.gz"):
+                if not f.endswith(".%s.rootfs.tar.gz" % selected_appID.lower()):
```

Comment:
```suggestion
                if not f.endswith(f".{selected_appID.lower()}.rootfs.tar.gz"):
```

F-strings are best-strings.

Reply:
Thanks! I never know about the recent python formatting! Picking your suggestion :)